docs/add-merge-automation-to-your-repository.md

```diff
@@ -11,31 +11,42 @@ In a nutshell: if tide is enabled then for any created PR that has `lgtm` and `a
 
 Example configuration PR to enable tide for KubeVirt [prow]: https://github.com/kubevirt/project-infra/pull/653/
 
-Example workflow:
-- tide is configured for PRs on a repository to require `lgtm` and `approved` labels and missing `needs-rebase` label
+## Example workflow
+
+Initial situation: `tide` is configured for PRs on a repository to require `lgtm` and `approved` labels and missing `needs-rebase` label
+
 - Alice creates a pull request, which triggers presubmit jobs to run for the PR
 - Some presubmit jobs have `optional: false` set, which makes their status required to be successful
 - All jobs succeed and add a success status to the commit object in GitHub
-- Note: tide does not consider this PR to be in the merge pool as it misses the `lgtm` and `approved` labels
+> [!NOTE]
+> `tide` does **not** consider this PR to be in the merge pool as it misses the `lgtm` and `approved` labels
 - Alice asks Bob and Charles for a review on the pull request
 - Bob (a member) reviews and adds a `/lgtm` comment
-- Note: a GitHub review automatically acts as if the `/lgtm` comment had been placed on the PR (see [config](https://github.com/kubevirt/project-infra/blob/main/github/ci/prow-deploy/kustom/base/configs/current/plugins/plugins.yaml#L463))
+> [!NOTE]
+> A GitHub review automatically _may_ act as if the `/lgtm` comment had been placed on the PR (see [config](https://github.com/kubevirt/project-infra/blob/fbb84ab7a4206079c94c1ee226a5af12915d9f0b/github/ci/prow-deploy/kustom/base/configs/current/plugins/plugins.yaml#L747))
 - After that the default branch gets updated by another merge, where merging Alice's PR would cause a conflict, which adds a `needs-rebase` label
-- Charles (a maintainer) reviews and adds an `/approve` comment
-- Note: tide does not consider this PR to be in the merge pool as it has the `needs-rebase` label
+- Charles - an approver listed in the [OWNERS] files for _all the files_ touched in the PR - reviews and adds an `/approve` comment
+> [!NOTE]
+> tide does **not** consider this PR to be in the merge pool as it has the `needs-rebase` label
```

Comment:
```suggestion
> `tide` does **not** consider this PR to be in the merge pool as it has the `needs-rebase` label
```

Reply:
There's mixed use of 'tide' and '`tide`' throughout. It is probably easier if you do a global replace than me suggesting changes here.